### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SimonMolinsky**
Please check how does it sound? Should I add other headers? Are there typos, ...

### CODE REVIEW

**Comment by Reckony** [comment]
LGTM :boom:

**Review by Reckony — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,47 @@
...
+3. I don't know.
+4. Rather agree.
+5. Strongly agree.
+
+
+```
+
+The order between categories makes analysis complex, and the fact that answers are polarized between opposing states. Moreover, border between categories is subjective and it depends from the person's experiences, feelings, and knowledge about a surveying topic.
+
+Classical measurements of central tendency do not fit well ordinal data [ADD BIBLIOGRAPHY]. That's why we encourage you to use `ordinal-scale-stats` package for your analysis of survey responses.
```
Comment (fgrabowski) [inline README.md:25]: I believe that there should be "do not fit well WITH ..."

Comment (fgrabowski) [inline README.md:25]: Also, "That's why we encourage you to use THE `ordinal-scale-stats` package for your analysis of survey responses."

File: README.md [hunk 2]
```diff
@@ -1 +1,47 @@
-# ordinal-scale-stats-py
\ No newline at end of file
+# Ordinal-Scale-Stats-py
+
+Python package that helps you analyze ordinal data.
+
+## Introduction
+
+Ordinal scale data is common. Companies and governments can quickly perform large-scale research with surveys.
+Usually, a survey output is placed on the Likert scale, where answers are ordered to describe person's feelings about survey's topic.
```
Comment (fgrabowski) [inline README.md:8]: about THE survey's topic

File: README.md [hunk 3]
```diff
@@ -1 +1,47 @@
...
+Typical example of a survey is when person is asked to agree with a statement with answers on a five-level scale:
```
Comment (fgrabowski) [inline README.md:9]: A typical

Comment (fgrabowski) [reply README.md:9]: and also "a person"

**Review by fgrabowski — CHANGES_REQUESTED** [review]
However, I am not an english teacher so I may be wrong at some points, but there are also points that I am sure of

File: README.md [hunk 4]
```diff
@@ -1 +1,45 @@
-# ordinal-scale-stats-py
\ No newline at end of file
+# Ordinal-Scale-Stats-py
+
+Python package that helps you analyze ordinal data.
+
+## Introduction
+
+Ordinal scale data is common. Companies and governments can quickly perform large-scale research with surveys.
```
Comment (fgrabowski) [inline README.md:7]: I think it should be,
"Ordinal scale data ARE common"
because word "data" is plural

**Review by fgrabowski — CHANGES_REQUESTED** [review]

**Review by fgrabowski — APPROVED** [review]

**Review by Arturstrag — APPROVED** [review]

**Pull request merged**